run-qemu/run.sh

```diff
@@ -5,6 +5,26 @@ trap 'exit 2' ERR
 
 source $(cd $(dirname $0) && pwd)/../helpers.sh
 
+foldable start bpftool_checks "Running bpftool checks..."
+bpftool_exitstatus=
+if [[ "${KERNEL}" = 'LATEST' ]]; then
+	# "&& true" does not change the return code (it is not executed if the
+	# Python script fails), but it prevents the trap on ERR set at the top
+	# of this file to trigger on failure.
+	"${REPO_ROOT}/${KERNEL_ROOT}/tools/testing/selftests/bpf/test_bpftool_synctypes.py" && true
+	bpftool_exitstatus=$?
+	if [[ "$bpftool_exitstatus" -eq 0 ]]; then
+		echo "bpftool checks passed successfully."
+	else
+		echo "bpftool checks returned ${bpftool_exitstatus}."
+	fi
+	bpftool_exitstatus="bpftool:${bpftool_exitstatus}"
+else
+	echo "bpftool checks skipped."
```

Comment:
adding a comment on why such check is not needed for non 'LATEST' case would be beneficial

Reply:
I agree. But I don't know why it isn't, to be honest :-) If you know let me know and I can add it.

Reply:
This test is to check that everything is consistent between the commands supported by bpftool, its known type names, its documentation, its bash completion. We want to keep that in sync in the kernel, but there's no point in checking it for older kernels. If it failed on older kernels, it might be that some item was missing from the docs, not that some feature was broken.

So we want to run it on LATEST to make sure that nobody introduces a new type without updating bpftool everywhere necessary, but there's no interest in running this script for older kernels, we won't detect bugs or breakages.